cltk/phonology/old_english/orthophonology.py

```diff
@@ -39,7 +39,7 @@
 sh = Consonant(Place.post_alveolar, Manner.fricative, Voiced.neg, 'ʃ')
 ch = Consonant(Place.palatal, Manner.fricative, Voiced.neg, 'ç')
 x  = Consonant(Place.velar, Manner.fricative, Voiced.neg, 'x')
-y  = Consonant(Place.velar, Manner.fricative, Voiced.pos, 'ɣ')
+xh  = Consonant(Place.velar, Manner.fricative, Voiced.pos, 'ɣ')

```

Comment:
In my opinion, gh is a better transcription, since it is a fractative g.

Reply:
True.  Changing.